docs/quickstart.rst

```diff
@@ -287,6 +298,12 @@ service.
 
     !IrisClassifier info
 
+You can use `docs` command to get all APIs in OpenAPI format.
+
+.. code-block:: python
+
+    !IrisClassifier docs
```

Comment:
could you also add the other form of the same CLI command here: ```bentoml docs {ARCHIVE_PATH}```

Reply:
And also add the same thing to the quick start jupyter notebook?

Reply:
Yes sure